packages/framework/aqueduct/src/container-runtime-factories/index.ts

```diff
@@ -3,5 +3,11 @@
  * Licensed under the MIT License.
  */
 
-export { BaseContainerRuntimeFactory } from "./baseContainerRuntimeFactory";
-export { ContainerRuntimeFactoryWithDefaultDataStore } from "./containerRuntimeFactoryWithDefaultDataStore";
+export {
+	BaseContainerRuntimeFactory,
+	type BaseContainerRuntimeFactoryProps,
+} from "./baseContainerRuntimeFactory";
+export {
+	ContainerRuntimeFactoryWithDefaultDataStore,
+	type ContainerRuntimeFactoryWithDefaultDataStoreProps,
+} from "./containerRuntimeFactoryWithDefaultDataStore";
```

Comment:
I wonder if these would benefit from a release notes entry along the lines of "Props for constructors of X and Y are now named types".